build.sbt

```diff
@@ -39,8 +39,8 @@ val commonSettings = Sonatype.sonatypeSettings ++ releaseSettings ++ Seq(
   organization := "com.spotify",
   name := "ratatool",
   description := "A tool for random data sampling and generation",
-  scalaVersion := "2.11.12",
-  crossScalaVersions := Seq("2.11.12", "2.12.10"),
+  scalaVersion := "2.12.10",
+  crossScalaVersions := Seq("2.12.10"),
```

Comment:
We should start compiling for 2.13 as well

Reply:
Hmm nvm it looks like scio is only compiling for 2.12 rn

Reply:
Actually they are, its just specified separately, likely differs by module. https://github.com/spotify/scio/blob/master/build.sbt#L441

Reply:
👍 will add

Reply:
lol yeah. I first thought it was only 2.12 as well.